### ISSUE TITLE
Provas de conceito

### ISSUE THREAD

**Issue by serralvo** [serralvo/mean-scrum#3]
Galera, acho que seria interessante criamos uma prova de conceito para algumas skills (MongoDB e AngularJS), eu por exemplo sei o básico de mongo e quase nada do menino de ouro do google, o que acham?

**Comment by dBiazioli** [comment]
Sim, otimizar as skills é essencial. Mongo nunca usei, Express, Angular e Node sei o básico tb...

**Comment by rafaelhgc** [comment]
A ideia é bacana, mas o que você sugere?

**Comment by serralvo** [comment]
Que tal um crud básico? Cada um trabalha em algo e depois subimos em uma branch específica para isso, o que acham?

**Comment by rafaelhgc** [comment]
Legal, vamos usar algo aleatório ou um exemplo que possa ser importado no projeto?

**Comment by serralvo** [comment]
Acho que podemos segmentar e depois tentar realizar a integração
